client/components/operations/SyncButton.js

```diff
@@ -29,25 +29,29 @@ export default class SyncButton extends React.Component {
     }
 
     render() {
-        let text = this.state.isSynchronizing
-                   ? <div className="last-sync">
-                        <span className="option-legend">
-                            {$t('client.operations.syncing')}
-                        </span>
-                        <span className="fa fa-refresh fa-spin"></span>
-                     </div>
-                   : <div className="last-sync">
-                        <span className="option-legend">
-                            {$t('client.operations.last_sync')}
-                            {' ' + new Date(this.props.account.lastChecked).toLocaleString()}
-                        </span>
-                        <a href='#' onClick={this.onFetchOperations.bind(this)}>
-                            <span className="option-legend fa fa-refresh"></span>
-                        </a>
-                    </div>;
+        let text = (
+            this.state.isSynchronizing ?
+                <div className="last-sync">
+                    <span className="option-legend">
+                        { $t('client.operations.syncing') }
+                    </span>
+                    <span className="fa fa-refresh fa-spin"></span>
+                </div> :
+                <div className="last-sync">
+                    <span className="option-legend">
+                        { $t('client.operations.last_sync') }
+                        { ` ${new Date(this.props.account.lastChecked).toLocaleString()}` }
```

Comment:
Instead of this, let's do: `&nbsp;` in raw html, then just the content of the template string.